stream-chat-android-ui-guides/detekt-baseline.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<SmellBaseline>
+  <ManuallySuppressedIssues/>
+  <CurrentIssues>
+    <ID>ComplexMethod:LoginUsers.kt$LoginUsers$fun createUsers(): List&lt;LoginUser></ID>
```

Comment:
@JcMinarro should we add special rules for guides? It seems most of them are Compose related 🤔

Reply:
I've added the Compose part of the config